Web/src/features/authentication/hooks/useAuthStatus.ts

```
@@ -39,13 +52,125 @@ export function useAuthStatus() {
     imageUrl: clerkUser.imageUrl || '',
   } : null, [clerkUser])
 
+import { useCallback, useMemo, useEffect, useState, useRef } from 'react'
+
+  const [retryCount, setRetryCount] = useState(0)
```

Biome (2.1.2)

[error] Shouldn't redeclare 'retryCount'. Consider to delete it or rename it.

'retryCount' is defined here:


(lint/suspicious/noRedeclare)